engines/drascula/graphics.cpp

```diff
@@ -80,8 +82,20 @@ void DrasculaEngine::moveCursor() {
 			color_abc(kColorRed);
 	} else if (!_menuScreen && _color != kColorLightGreen)
 		color_abc(kColorLightGreen);
-	if (_hasName && !_menuScreen)
+	if (_hasName && !_menuScreen) {
+		if (_previousSaid != textName) {
```

Comment:
It is the same problem as in another, more recent PR. You need to check here that TTS is pronouncing text still, and if it is not saying, then still say the phrase. Otherwise, if two consequent request have the same phrase, e.g. when a player clicks on the same item twice in a row, the second phrase will be not pronounced.